### PR TITLE
Implement side-by-side blueprints.

### PR DESCRIPTION
**Pull request by mshima**
Fixes https://github.com/jhipster/generator-jhipster/issues/11173.

-   Please make sure the below checklist is followed for Pull Requests.

-   [ ] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
-   [ ] Tests are added where necessary
-   [ ] Documentation is added/updated where necessary
-   [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

**Comment by mshima** [comment]
@murdos what do you think about this PR?

File: generators/generator-base-private.js [hunk 1]
```diff
@@ -815,11 +815,15 @@ module.exports = class extends Generator {
                     jhipsterContext: this
                 };
                 this.useBlueprint = true;
-                this.composeExternalModule(blueprint, subGen, finalOptions);
+                const generetor = this.composeExternalModule(blueprint, subGen, finalOptions);
```
Comment (pascalgrimaud) [inline generators/generator-base-private.js]: typo: generator (instead of generetor)

File: test/blueprint/sbs-blueprint.spec.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+const constants = require('../../generators/generator-constants');
+
+const ANGULAR = constants.SUPPORTED_CLIENT_FRAMEWORKS.ANGULAR;
+
+const mockBlueprintSubGen = class extends Generator {
+    constructor(args, opts) {
+        super(args, opts);
+        const jhContext = (this.jhipsterContext = this.options.jhipsterContext);
+        if (!jhContext) {
+            this.error("This is a JHipster blueprint and should be used only like 'jhipster --blueprint myblueprint')}");
```
Comment (pascalgrimaud) [inline test/blueprint/sbs-blueprint.spec.js]: should be blueprints (with 's')

File: generators/generator-base-private.js (hunk 1 above)
Comment (mshima) [reply generators/generator-base-private.js]: Thanks

File: test/blueprint/sbs-blueprint.spec.js (hunk 2 above)
Comment (mshima) [reply test/blueprint/sbs-blueprint.spec.js]: Copy/paste problem.
Updated every test description with same error.

**Comment by mshima** [comment]
@pascalgrimaud done.

**Pull request merged**